Translation/en/Text/Main/ActionScene.txt

```diff
@@ -90,8 +103,8 @@ r:^※コイカツポイントを(\d+)pt消費します。$=※ Consumes $1 Koik
 お泊り=Staying
 お祈り=Prayer
 所持数=Num Items
-滞在先=Accommodations
-現在地=Location
+//滞在先=Bookings
+//現在地=Place
```

Comment:
Why comment these out?

Reply:
They are repeated above, and keeping them commented makes easier to know what is insice `/ActionScene/UI/PlayerStatus`. Since GeBo organized by path I think he have other plans for the file.